virtool/jobs/runner.py

```diff
@@ -80,116 +80,3 @@ async def _get_job_from_lists(self):
 
     async def _run_job(self):
         pass
-
-
-class JobAgent(Base):
```

Comment:
Since `Base` is only subclassed by `JobRunner` now, `JobRunner` and `Base` can be merged into a single class.